qulice-findbugs/src/test/java/com/qulice/findbugs/FindBugsValidatorTest.java

```diff
@@ -54,6 +55,30 @@ public void passesCorrectFilesWithNoExceptions() throws Exception {
     }
 
     /**
+     * FindbugsValidator can report incorrectly added throws.
+     * @throws Exception If something wrong happens inside
+     */
+    @Test(expected = ValidationException.class)
+    public void reportsIncorrectlyAddedThrows() throws Exception {
```

Comment:
@krzyk can we add one more test ?
to make sure FindbugsValidator doesn't report correctly added throws